test/validation/api/ipsec/reass_test_vectors.h

```diff
@@ -391,5 +63,4 @@ reass_test_vectors_init(void)
 	test_vector_payload_populate(&pkt_ipv4_udp_p2_f3, false);
 	test_vector_payload_populate(&pkt_ipv4_udp_p2_f4, false);
 }
-
```

Comment:
This white space removal is unnecessary (maybe accidental).

Reply:
Ack.